samples/FtpServer_Files/app/application.cpp

```diff
@@ -28,11 +31,14 @@ void connectFail(const String& ssid, MacAddress bssid, WifiDisconnectReason reas
 
 void init()
 {
-	spiffs_mount(); // Mount file system, in order to work with files
-
 	Serial.begin(SERIAL_BAUD_RATE); // 115200 by default
```

Comment:
@mikee47 I know that this is not something that you have added but would it be possible to remove also the unnecessary comment from line 26 ?
```
// .. some you code for configuration ..
```